src/test/rustdoc-js-std/parser-generics.js

```diff
@@ -0,0 +1,37 @@
+const QUERY = ['A<B<C<D>,  E>', 'p<> u8'];
+
+const PARSED = [
+    {
+        elems: [],
+        foundElems: 0,
+        original: 'A<B<C<D>,  E>',
+        returned: [],
+        typeFilter: -1,
+        userQuery: 'a<b<c<d>,  e>',
+        error: 'Unexpected `<` after `<`',
+    },
+    {
+        elems: [
+            {
+                name: "p",
+                fullPath: ["p"],
+                pathWithoutLast: [],
+                pathLast: "p",
+                generics: [],
+            },
+            {
+                name: "u8",
+                fullPath: ["u8"],
+                pathWithoutLast: [],
+                pathLast: "u8",
+                generics: [],
+            },
+        ],
+        foundElems: 2,
+        original: "p<> u8",
```

Comment:
I notice that the generics eBNF says that there should be an error when the list is empty, but this test case says there isn't an error.

    generics = OPEN-ANGLE-BRACKET *WS nonempty-arg-list-without-generics *WS CLOSE-ANGLE-BRACKET

Which one is right?

Reply:
Needs to update the eBNF.

Reply:
Done!